File: lib/src/main/java/de/tum/in/tumcampusapp/models/Tuition.java
package de.tum.in.tumcampusapp.models;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

/**
 * Class holding tuition information.
 * <p/>
 * Note: This model is based on the TUMOnline web service response format for a
 * corresponding request.
 */
@Root(name = "row", strict = false)
public class Tuition {

    @Element(name = "frist")
    private String frist = "";

    @Element(name = "semester_bezeichnung")
    private String semester_bezeichnung = "";

    @Element(name = "soll")
    private String soll = "";

    public String getFrist() {
        return frist;
    }

    public String getSemesterBez() {
        return semester_bezeichnung;
    }

    public String getSoll() {
        return soll;
    }

}

File: app/src/main/java/de/tum/in/tumcampusapp/models/tumo/Tuition.kt
package de.tum.`in`.tumcampusapp.models.tumo

import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

/**
 * Class holding tuition information.
 *
 *
 * Note: This model is based on the TUMOnline web service response format for a
 * corresponding request.
 */
@Root(name = "row", strict = false)
data class Tuition(@field:Element(name = "frist")
                   var frist: String = "",
                   @field:Element(name = "semester_bezeichnung")
                   var semesterBez: String = "",
                   @field:Element(name = "soll")
                   var soll: String = "")
